superset-frontend/src/explore/components/QueryAndSaveBtns.jsx

```diff
@@ -91,37 +77,51 @@ export default function QueryAndSaveBtns({
     </Button>
   );
 
+  const theme = useTheme();
+
   return (
-    <Styles>
-      <div>
-        <ButtonGroup className="query-and-save">
-          {qryOrStopButton}
-          <Button
-            buttonStyle="tertiary"
-            buttonSize="small"
-            data-target="#save_modal"
-            data-toggle="modal"
-            disabled={saveButtonDisabled}
-            onClick={onSave}
-            data-test="query-save-button"
+    <div
+      css={{
```

Comment:
this is fine by me, but curious as to why you prefer the `<div css={{...` approach over the `styled.div` approach. They both seem to meet the same goal, so just wondering :D

Reply:
For me the use of a property seems more natural to the objective of customizing a component. [Here](https://benmcmahen.com/why-i-choose-emotion/#:~:text=The%20styled%20method%20uses%20the,to%20create%20buttons%20with%20styles.&text=With%20examples%20like%20this%2C%20it's,d%20use%20the%20style%20prop.) you will find a detailed explanation but to summarize:

- You’re still writing regular React components
- Object styles are easier to work with
- Naming things is hard
- You colocate styles with elements
- Composition is dead easy